Create for-wades-review.md (#2)
Cerchie/js-client-demo-script: README.md

          );
```

> Lucia: "Let's review the code I wrote to make this all happen. It starts with a POST request to my express route. I've attached an event listener to my buttons that fires this request on click. The data, including information about the vote [1], the name of the question, [2] and the timestamp [3] is sent to my route. I'm including the timestamp here in case I want to use stream processing like windowing in a later iteration of the project."

Author: ```suggestion
Lucia: "Let's review the code I wrote to make this all happen. It starts with a POST request to my express route. I've attached an event listener to my buttons that fires this request on click. The data, including information about the vote [1], the name of the question, [2] and the timestamp [3] is sent to my route. I'm including the timestamp here in case I want to use stream processing, say, creating a windowed filter, in a later iteration of the project."
```

}
```

> Lucia: "Now, in my Kafka consumer, I am parsing the incoming message which holds the vote state, [2] filtering for the last clicked message, creating variables to hold the current [3] question_id and [4] count, and [5] then sending the message to my websocket. You might notice that the syntax here is slightly different from the syntax in the first consumer. That's because this consumer uses the promisified API, versus the first consumer, which uses the API that avails itself of the features in node-rdkafka. This [6] `consumer.run` method from the promisified API feedds our function one message at a time using the [7] `eachMessage` handler."

Author: ```suggestion
Lucia: "Now, in my Kafka consumer, I am parsing the incoming message which holds the vote state, [2] filtering for the last clicked message, creating variables to hold the current [3] question_id and [4] count, and [5] then sending the message to my websocket. You might notice that the syntax here is slightly different from the syntax in the first consumer. That's because this consumer uses the promisified API, versus the first consumer, which uses the API that avails itself of the features in node-rdkafka. This [6] `consumer.run` method from the promisified API feeds our function one message at a time using the [7] `eachMessage` handler."
```

> Lucia: "Now, in my Kafka consumer, I am parsing the incoming message which holds the vote state, [2] filtering for the last clicked message, creating variables to hold the current [3] question_id and [4] count, and [5] then sending the message to my websocket. You might notice that the syntax here is slightly different from the syntax in the first consumer. That's because this consumer uses the promisified API, versus the first consumer, which uses the API that avails itself of the features in node-rdkafka. This [6] `consumer.run` method from the promisified API feedds our function one message at a time using the [7] `eachMessage` handler."

Reviewer: ```suggestion
Lucia: "Now, in my Kafka consumer, I am parsing the incoming message which holds the vote state, [2] filtering for the last clicked message, creating variables to hold the current [3] question_id and [4] count, and [5] then sending the message to my websocket. You might notice that the syntax here is slightly different from the syntax in the first consumer. That's because this consumer uses the promisified API, versus the first consumer, which uses the API that avails itself of the features in node-rdkafka. This [6] `consumer.run` method from the promisified API feeds our function one message at a time using the [7] `eachMessage` handler."
```

        );

        [4] let labelForBar = document.getElementById(`${progress_bar_id}-label`);

>          let max = Object.values(message.message.count)[0] + Object.values(message.message.count)[1];

Author: ```suggestion

```

>          let max = Object.values(message.message.count)[0] + Object.values(message.message.count)[1];

Reviewer: Looks like this line might be duplicated.
